velox/exec/fuzzer/ReferenceQueryRunner.h

```diff
@@ -24,10 +24,13 @@ namespace facebook::velox::exec::test {
 /// Query runner that uses reference database, i.e. DuckDB, Presto, Spark.
 class ReferenceQueryRunner {
  public:
-  enum class RunnerType { kPrestoQueryRunner, kDuckQueryRunner };
+  explicit ReferenceQueryRunner(memory::MemoryPool* rootPool)
+      : rootPool_(rootPool) {}
 
   virtual ~ReferenceQueryRunner() = default;
 
+  enum class RunnerType { kPrestoQueryRunner, kDuckQueryRunner };
```

Comment:
Any particular reason to move this class?

Reply:
Removed this change. Thanks.